app_dart/lib/src/request_handlers/reset_luci_task.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:cocoon_service/cocoon_service.dart';
+import 'package:cocoon_service/src/model/luci/buildbucket.dart';
+import 'package:meta/meta.dart';
+
+import '../datastore/cocoon_config.dart';
+import '../model/luci/buildbucket.dart' as bb;
+import '../request_handling/api_request_handler.dart';
+import '../request_handling/authentication.dart';
+import '../request_handling/body.dart';
+import '../service/datastore.dart';
+
+@immutable
+class ResetDevicelabTask extends ApiRequestHandler<Body> {
+  const ResetDevicelabTask(
+    Config config,
+    AuthenticationProvider authenticationProvider, {
+    @visibleForTesting DatastoreServiceProvider datastoreProvider,
+    @required this.buildBucketClient,
+  })  : datastoreProvider =
+            datastoreProvider ?? DatastoreService.defaultProvider,
+        super(config: config, authenticationProvider: authenticationProvider);
+
+  final DatastoreServiceProvider datastoreProvider;
+  final BuildBucketClient buildBucketClient;
+
+  static const String luciBuilderNameParam = 'LuciBuilderName';
+  static const String commitShaParam = 'CommitSha';
+
+  @override
+  Future<Body> post() async {
+    checkRequiredParameters(<String>[luciBuilderNameParam, commitShaParam]);
+    final String luciBuilderName = requestData[luciBuilderNameParam] as String;
+    final String commitSha = requestData[commitShaParam] as String;
+    await buildBucketClient.scheduleBuild(bb.ScheduleBuildRequest(
+      builderId: bb.BuilderId(
```

Comment:
Not sure which values are required here. It looks like buildset is usually something like:

`commit/git/fdc6f38a440472a696c5347515d89fe4dd9d699f`

Reply:
Didn't look into details before, but find some related codes:
https://github.com/flutter/cocoon/blob/17dda0d7bfa89ae83c2d1f15c64ce90272dc7876/app_dart/lib/src/model/luci/buildbucket.dart#L401
https://github.com/flutter/cocoon/blob/dbd113890d14171bfda5b55bdc7b3e104ce7e7cf/app_dart/lib/src/request_handlers/github_webhook.dart#L204
https://github.com/flutter/cocoon/blob/dbd113890d14171bfda5b55bdc7b3e104ce7e7cf/app_dart/lib/src/request_handlers/luci_status.dart#L67